README.md

```diff
@@ -0,0 +1,74 @@
+<h1 align="center" style="margin:1em;">
```

Comment:
Must the first elements of the readme be centre-aligned? I don't see there was any problem with everything being left-aligned as in the original. Centre-aligning the title and first section makes for an awkward disconnect when we get to the first section that isn't centre-aligned.

Whether or not the first elements *should* be centre-aligned or not, it's bad form to mix styling and content as is being done here.